lib/screens/StartupScreen.dart

```diff
@@ -840,19 +635,490 @@ class _StartPageState extends State<StartPage> {
+                            textInputAction: TextInputAction.done,
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  SizedBox(
+                    width: MediaQuery.of(context).size.width,
+                    child: Text(
+                      errorMessage,
+                      style: const TextStyle(fontSize: 14, color: Colors.red),
+                      textAlign: TextAlign.center,
+                    ),
+                  ),
+                  Container(
+                    padding: const EdgeInsets.only(top: 10, bottom: 5),
+                    width: MediaQuery.of(context).size.width,
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: <Widget>[
+                        SizedBox(
+                          width: 85,
+                          height: 36,
+                          child: ElevatedButton(
+                            onPressed: () async {
+                              if (areFieldsValid()) {
+                                String username = _email.value.text.trim();
+                                String password = _password.value.text.trim();
+                                String firstName = _firstName.value.text.trim();
+                                String lastName = _lastName.value.text.trim();
+
+                                try {
+
+                                  String payload = '{"firstName": "$firstName","lastName": "$lastName","username": "$username","password": "$password"}';
+                                  final ret = await Authentication.register(payload);
+                                  if (status(ret)) {
+                                    var data = json.decode(ret.body);
+                                    user = UserData(
+                                      firstName: data['userInfo']['firstName'],
+                                      lastName: data['userInfo']['lastName'],
+                                      email: data['userInfo']['username'],
+                                      lastSeen: data['userInfo']['lastSeen'],
+                                      accessToken: data["accessToken"],
+                                      refreshToken: data["refreshToken"],
+                                      password: password,
+                                    );
+
+                                      errorMessage = '';
+                                      clearFields();
+                                      //TODO Add support for verifying registered email
+                                      /*setState(() {
+                                        state = 1;
+                                      });*/
+                                      Navigator.pushNamedAndRemoveUntil(context,
+                                          '/food', ((Route<dynamic> route) => false));
```

Comment:
The alignment is messed up here, there are different indentation levels in the same block